packages/pangea-node-sdk/tests/integration/audit.test.ts

```diff
@@ -30,22 +32,23 @@ const JSON_OLD_DATA = {
   ct6: "cm6",
 };
 
-const environment = TestEnvironment.LIVE;
-const token = getTestToken(environment);
+const environment = TestEnvironment.DEVELOP;
+const tokenGeneral = getTestToken(environment);
 const tokenVault = getVaultSignatureTestToken(environment);
+const tokenMultiConfig = getMultiConfigTestToken(environment);
 const domain = getTestDomain(environment);
 const config = new PangeaConfig({ domain: domain, customUserAgent: "sdk-test" });
-const audit = new AuditService(token, config);
+const auditGeneral = new AuditService(tokenGeneral, config);
 const auditVault = new AuditService(tokenVault, config);
-const auditWithTenantId = new AuditService(token, config, "mytenantid");
+const auditWithTenantId = new AuditService(tokenGeneral, config, "mytenantid");
 
 it("log an audit event. no verbose", async () => {
   const event: Audit.Event = {
     actor: ACTOR,
     message: MSG_NO_SIGNED,
     status: MSG_NO_SIGNED,
   };
-  const response = await audit.log(event);
+  const response = await auditGeneral.log(event);
```

Comment:
what does audit general mean?

Reply:
I called that way to just not be `audit` due to we have `auditCustomSchema`, `auditWithTenantId`, `auditVault`